### PR TITLE
Add android support

### PR DESCRIPTION
**Pull request by maluramichael**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,10 +41,17 @@ af.init(appId, devKey, function(err, events) {
     af.sendTrackingWithEvent(eventName, {key1: value1, key2: value2}, function(err, events) {
         //events is {key1: value1, key2, value2}
     });
+})
+```
 
-    //To get appsFlyerUID an event,
-    af.getAppsFlyerUID(function(err, uid) {
```
Comment (jeanpaulgorman) [inline README.md:33]: The af.getAppsFlyerUID does work in the IOS version, so can you maintain the method call under the IOS example. Unless you're able to add support for the af.getAppsFlyerUID method to Android?

Comment (maluramichael) [inline README.md:33]: Oh dang. This was my bad. I removed a bit too much of the usage. getAppsFlyerUID is still working in ios but not yet supported in my android with my changes.

**Comment by maluramichael** [comment]
Would be nice if some people could try this branch in their app and let me know if something does not work as expected.

**Comment by ppsreejith** [comment]
Hey, thank you for this :) . This would definitely be very useful.

**Pull request merged**